automation_tools/scripts/ga-migration/main.py

```diff
@@ -96,8 +96,8 @@ def migrate_repo(path):
 
     # Upgrade Sphinx 1 to 3 in setup.py
     replace_regex(
-        r"Sphinx>=1.[0-9](.[0-9])?.*,",
-        "Sphinx>=3',",
+        r"Sphinx>=1.[0-9](.[0-9])?",
```

Comment:
I found packages that had `Sphinx>1.x,<2.x`, I think this PR would generate `Sphinx>=3,<2.x`, which is invalid.

Reply:
https://regex101.com/r/TtS6F3/1